package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "markdown-linting",
```

Comment:
We don’t need to specify attributes like `name`, `version`, `description` etc. if we don’t plan to treat this folder as an npm package (i.e. we don’t need to import from it or publish it to the registry). We can go for `"private": true` instead – that’s enough to silence npm/yarn warnings about missing metadata.